src/rating/styled-components.js

```diff
@@ -0,0 +1,120 @@
+    const sadRatingSVG = `<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="22" cy="22" r="20" fill="${encodeURIComponent(
+      emoticonFill,
+    )}"/><ellipse cx="16" cy="18" rx="2" ry="2" fill="black"/><ellipse cx="28" cy="18" rx="2" ry="2" fill="black"/><path fill-rule="evenodd" clip-rule="evenodd" d="M21.8377 28C24.8073 28 26.6585 29.7697 27.1609 30.5442C27.4615 31.0075 28.0807 31.1395 28.5441 30.839C29.0074 30.5384 29.1394 29.9192 28.8388 29.4558C28.0439 28.2303 25.614 26 21.8377 26C18.0453 26 15.8091 28.25 15.1202 29.5245C14.8576 30.0103 15.0385 30.6171 15.5244 30.8797C16.0102 31.1423 16.617 30.9614 16.8796 30.4755C17.2718 29.75 18.8842 28 21.8377 28Z" fill="black"/></svg>`;
+    const neutralRatingSVG = `<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="22" cy="22" r="20" fill="${encodeURIComponent(
+      emoticonFill,
+    )}"/><ellipse cx="16" cy="19" rx="2" ry="2" fill="black"/><ellipse cx="28" cy="19" rx="2" ry="2" fill="black"/><path fill-rule="evenodd" clip-rule="evenodd" d="M15 28C15 27.4477 15.4477 27 16 27L16 28L16 29C15.4477 29 15 28.5523 15 28ZM28 28L28 29H16L16 28L16 27H27.9995L28 28ZM28 28L28 29C28.5523 29 29 28.5523 29 28C29 27.4477 28.5518 27 27.9995 27L28 28Z" fill="black"/></svg>`;
+    const happyRatingSVG = `<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="22" cy="22" r="20" fill="${encodeURIComponent(
+      emoticonFill,
+    )}"/><ellipse cx="16" cy="18" rx="2" ry="2" fill="black"/><ellipse cx="28" cy="18" rx="2" ry="2" fill="black"/><path fill-rule="evenodd" clip-rule="evenodd" d="M21.8377 29C24.8073 29 26.6585 27.2303 27.1609 26.4558C27.4615 25.9925 28.0807 25.8605 28.5441 26.161C29.0074 26.4616 29.1394 27.0808 28.8388 27.5442C28.0439 28.7697 25.614 31 21.8377 31C18.0453 31 15.8091 28.75 15.1202 27.4755C14.8576 26.9897 15.0385 26.3829 15.5244 26.1203C16.0102 25.8577 16.617 26.0386 16.8796 26.5245C17.2718 27.25 18.8842 29 21.8377 29Z" fill="black"/></svg>`;
+    const veryHappyRatingSVG = `<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="22" cy="22" r="20" fill="${encodeURIComponent(
```

Comment:
How about we move all the SVGs in a single separate file?

Reply:
I've refactored these into a shared utility file